uwsgi/admin.wsgi

```diff
@@ -43,13 +43,17 @@ cache.make_cache("blob_schema", 50, 24 * 60 * 60)
 
 dbo.setDb(os.environ["DBURI"])
 if os.environ.get("NOTIFY_TO_ADDR"):
+    tls = False
```

Comment:
I would probably use `use_tls` or `tls_enabled`; `tls` sounds more like an object name to me, not a boolean.